tests/integration/test_core.py

```diff
@@ -133,3 +109,78 @@ async def test_wordpress_functionality(
             admin_username="admin",
             admin_password=default_admin_password
         )
+
+
+@pytest.mark.asyncio
+async def test_wordpress_default_themes(
+        unit_ip_list,
+        get_theme_list_from_ip
+):
+    """
+    arrange: after WordPress charm has been deployed and db relation established
+    act: check installed WordPress themes
+    assert: All default themes should be installed
```

Comment:
Very picky, should `All` be lower case here?

Reply:
Fixed, thanks